src/brokers/dkb.js

```diff
@@ -168,41 +162,34 @@ export const canParsePage = (content, extension) =>
   (isBuy(content) || isSell(content) || isDividend(content));
 
 export const parsePages = pages => {
-  let type, date, time, isin, company, shares, price, amount, fee, tax;
   const firstPage = pages[0];
 
+  let type, amount, price, date, time;
+  const stueckIdx = firstPage.findIndex(t => t.includes('Stück'));
```

Comment:
```suggestion
  const pieceIdx = firstPage.findIndex(t => t.includes('Stück'));
```

And other references of `stueckIdx`

Reply:
Fixed